ncbi/src/main/java/org/reactome/release/Main.java

```diff
@@ -0,0 +1,104 @@
+	 * generates export files.  The files generated are:
+	 * NCBI Gene XML
+	 *     "Link" XML nodes describing NCBI Gene Identifier relationships to either UniProt entries or Top Level
+	 *     Pathways in Reactome
+	 * NCBI Gene Protein File (not uploaded to NCBI)
+	 *     Tab delimited file of UniProt accession to NCBI Gene identifiers in Reactome
+	 * NCBI Protein File
+	 *     Entries of all UniProt accessions in Reactome associated with an NCBI Gene Identifier (file only contains
+	 *     UniProt accessions)
+	 * UCSC Entity File
+	 *     Entires of all human, rat, and mouse UniProt accessions in Reactome
+	 * UCSC Entity File
+	 *     UniProt accessions (human, rat, and mouse) mapped to the events in which they participate in Reactome
+	 * Europe PMC Profile File
+	 *     Short XML file identifying Reactome as a data provider to Europe PMC
+	 * Europe PMC Link File
+	 *     "Link" XML nodes describing Reactome Pathways connected to PubMed literature references
+	 * @param args Command line arguments for the post-release data files export
+	 * @throws IOException Thrown if unable to create output directory or write files
+	 */
+	public static void main( String[] args ) throws IOException {
+		logger.info("Beginning NCBI, UCSC, and Europe PMC export step...");
+
+		String pathToResources =
+			args.length > 0 ?
+			args[0] :
+			Objects.requireNonNull(
+				Main.class.getClassLoader().getResource("sample_config.properties")
+			).getPath();
+
+		Properties props = new Properties();
+		try {
+			props.load(new FileInputStream(pathToResources));
+		} catch (IOException e) {
+			e.printStackTrace();
+		}
+		int version = Integer.parseInt(props.getProperty("reactomeVersion", "67"));
+		String outputDir = props.getProperty("outputDir", "archive");
+		Files.createDirectories(Paths.get(outputDir));
+		logger.info("Files for Reactome version " + version + " will be output to the directory " + outputDir);
+
+		Session graphDBSession = getGraphDBDriver(props).session();
+
+		List<NCBIEntry> ncbiEntries = NCBIEntry.getUniProtToNCBIGeneEntries(graphDBSession);
+
+		// Write NCBI Gene related Protein File
+		NCBIGene.getInstance(ncbiEntries, outputDir, version).writeProteinFile();
+
+		// Write NCBI Gene Files (split into multiple files to conform with 15MB upload maximum)
+		int numGeneXMLFiles = Integer.parseInt(props.getProperty("numGeneXMLFiles", "1"));
+		NCBIGene.getInstance(ncbiEntries, outputDir, version).writeGeneXMLFiles(
+			graphDBSession,
+			numGeneXMLFiles
+		);
+
+		// Write NCBI Protein File
+		NCBIProtein.getInstance(ncbiEntries, outputDir, version).writeNCBIProteinFile();
+
+		// Write UCSC Entity and Event Files
+		UCSC.getInstance(outputDir, version).writeUCSCFiles(graphDBSession);
+		// Write Europe PMC Profile and Link Files
+		EuropePMC.getInstance(outputDir, version).writeEuropePMCFiles(graphDBSession);
+
+		graphDBSession.close();
+		logger.info("Finished NCBI, UCSC, and Europe PMC export step");
+
+		System.exit(0);
```

Comment:
I think 0 is a default exit code, so I'm not sure this is necessary. On the other hand, I don't think it will cause any problems...

Reply:
Yes, I think it is.  I tried just `System.exit()` but that method doesn't exist.  I added an explicit exit call here because the program just seemed to continue running in the IDE without it.

Reply:
Really? That's strange. Might be an issue with the IDE? Does it do that when you run it from the command line?

Reply:
Seems to happen on the command line as well.

Reply:
That's... weird. Probably warrants further investigation.